.github/workflows/test_pytorch_wheels.yml

```
@@ -132,4 +134,4 @@ jobs:
       - name: Run PyTorch tests
         if: "contains(inputs.test_runs_on, 'linux')"
         run: |
-          bash external-builds/pytorch/run_linux_pytorch_tests.sh
+          ./external-builds/pytorch/run_linux_pytorch_tests.py
```

nit: I'd like to use the same workflow code on Windows (once https://github.com/ROCm/TheRock/issues/999 is fixed), and there we should be running Python scripts through an explicit launcher and not rely on the shebang within the "executable" file.
```suggestion
     - name: Run PyTorch tests
if: "contains(inputs.test_runs_on, 'linux')"
run: |
          python3 ./external-builds/pytorch/run_linux_pytorch_tests.py
```